src/pages/components/Navbar.tsx

```diff
@@ -75,8 +75,9 @@ export function Navbar() {
         </ul>
 
         {/* Mobile */}
-        <label className="btn swap btn-circle swap-rotate border-none bg-transparent hover:bg-transparent lg:hidden">
+        <label className="swap-rotate btn swap btn-circle border-none bg-transparent hover:bg-transparent lg:hidden">
           <input
+            aria-label="Open Menu"
```

Comment:
Add a label on the hamburger icon when opening on the small screen.
![](https://user-images.githubusercontent.com/8603085/191321127-a12c5d5a-be69-4ebd-82d4-6f7ca85c29ab.png)